src/stratigraphy/util/layer_identifier_column.py

```diff
@@ -16,7 +40,7 @@ def __init__(self, entries: list[TextWord]):
         Args:
             entries (list[TextWord]): The entries corresponding to the layer indices.
```

Comment:
Maybe rename this argument to "words", to clarify that at this point, we don't have actual "LayerIdentifierEntry" objects yet.